ansible/roles/uan_interfaces/defaults/main.yml

```diff
@@ -42,6 +42,7 @@ uan_user_access_cfg: 'CAN'
 uan_can_bond_slaves: 
   - "ens10f1"
   - "ens1f1"
+uan_chn_device: "hsn0"
```

Comment:
Should this be added to the README.md for `uan_interfaces`?

Reply:
Ah, yes!  Thanks for catching that!